var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -71,12 +74,15 @@ def install(self, spec, prefix):
     @run_after('install')
     @on_package_attributes(run_tests=True)
     def test(self):
+        import os
```

Comment:
Imports should be at the top of the package

Reply:
no prob, just localize it to the point of usage (only needed in tests).
will move back up